### ISSUE TITLE
Set initial window size in single-window mode - currently default window is too large.

### ISSUE THREAD

**Issue by jedokaplan** [OSGeo/grass#2600]
**Description**
The initial window size in "single-window" mode is too large - the window width covers both of my two-screen setup.

**To Reproduce**
1. Launch GRASS.
2. In Settings, choose "single-window" mode.
3. Restart GRASS.
4. See that the GRASS window upon open occupies the full screen (or screens).

**Expected behavior**
It would be great if the initial window size would be remembered between sessions, or allow the user to set the window size in preferences.

**System description (please complete the following information):**

- Operating System: Linux CentOS 7
- GRASS GIS version 8.2.0

**Additional context**

I have tried changing the default settings in `.grass8/wx.json` as below, but these do not seem to have any effect.

```
  "general": {
    "defWindowPos": {
      "enabled": true,
      "dim": "1600,100,1280,1024"
    },
```

**Comment by nilason** [comment]
I can confirm this on both main and for v8.2.0 with Mac.
The fix #2417 to the somewhat related issue #2415 did not alter *this* specific problem in any way.

**Comment by nilason** [comment]
To be specific: the stored position (general:defWindowPos:dim) in wx.json is seemingly ignored at gui startup.

**Comment by tmszi** [comment]
> To be specific: the stored position (general:defWindowPos:dim) in wx.json is seemingly ignored at gui startup.

Window position and size is loaded and set correctly but it is overrided by `Fit()` method (line 192).
https://github.com/OSGeo/grass/blob/daaf4cb9b92b5e3263d4e84a01bb513536a1b6f8/gui/wxpython/main_window/frame.py#L187-L192

**Comment by tmszi** [comment]
How to save user defined wxGUI window size and position as default (require merged PR #2619):

1. Launch wxGUI with single window mode (require
2. Interactively resize wxGUI window or change window position
3. Open wxGUI settings via menu Settings -> Preferences
4. On the General page (tab) from the Workspace setting **check Save current windows layout as default**
5. Hit Preferences dialog Save button
6. Close wxGUI
7. Launch wxGUI and check the window position and size

If you want to change the wxGUI position and size again, you need to open the wxGUI preferences dialog again and press the Save button again.
